### DISCUSSION TITLE
CS50 - 哈佛大學免費線上課程

### DISCUSSION THREAD

**Discussion by a45320614** [1101-GS4719-C/Python-Community-GS4719-C#881] [General]
CS50是美國哈佛大學所開設的一系列免費線上程式課程，包含了最熱門的程式入門課程CS50x，還有教你製作遊戲的CS50G或是專為律師及法律系學生所設計的CS50L等等，超酷。

CS50x每個禮拜都會上一個主題，有C, Python, SQL, HTML, CSS, JavaScript等等，也有資料結構、演算法、資訊安全這樣的Computer Science基礎課程，總共有11個禮拜，官網建議的每周學習時間是10\~20小時，大家可以根據自己的能力安排進度~

[CS50x](https://pll.harvard.edu/course/cs50-introduction-computer-science?delta=0)

**Comment by StevenWanglolz** [comment]
我已經申請加入那個課程了，希望我真的會騰出時間上那些課還有寫它的作業哈哈哈

**Comment by Chiway1229** [comment]
哇 我前幾天才報名，沒想到就看到相關介紹了
希望能夠滿載而歸

**Comment by Yunhans** [comment]
還能順便練英文聽力真的讚

**Comment by Kotori0501** [comment]
感覺報名能學到很多
但我不知道有沒有時間可以參加
希望我能騰出一個時間好好學習

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
不但能學到相關知識還能練到英文聽力 真是一石二鳥

**Comment by ruthchen1020** [comment]
會持續關注這樣的課程的，雖然我現在完全沒有時間去做這個課程，但未來可以成為日後進修的工具!
感謝分享!

**Comment by peggy0329** [comment]
感謝分享資源！！不過每週學習時間好久🤧
對我一個沒有程式基礎的人，用全英文上課感覺好吃力哈哈哈
不過未來學習較多後，可以善用這個資源！

**Comment by CPCpyt** [comment]
感謝分享資源！不過每周學習時間真是驚人！
而且我一定要更久...心有而餘力不足...

**Comment by Tuoresa** [comment]
感謝分享！
現在的網際網路真的都有好多資源能獲得
不抓緊就很可惜的感覺

這課程學習量會被大量的知識淹沒
希望我能盡快騰出時間申請帳號參與！
